style/icons/shield40_us_ar_2.svg

```diff
@@ -0,0 +1,4 @@
+<svg width="21" height="20" version="1.0" xmlns="http://www.w3.org/2000/svg">
+ <path style="stroke:#000;stroke-width:2;stroke-miterlimit:4" d="M 3.758282,16.35231 H 1.908052 V 6.54654 L 1.164362,1.00012 h 16.8806 c 0.44463,0 0.538351,0.6154577 0.38277,1.11034 -0.178366,0.5673582 -0.393809,1.1957 -0.07685,1.1957 h 1.62779 v 1.03656 c 0,0.1424368 -0.02772,0.2924598 -0.0577,0.38849 l -1.03254,3.3078 c -0.0996,0.3190811 -0.12015,0.3383476 -0.31259,0.72438 l -2.12654,4.26582 c -0.295646,0.593063 -0.354434,1.272592 -0.37604,1.7596 -0.02611,0.588508 0.285034,1.204005 0.28312,1.753311 l -0.006,1.722079 C 16.349286,18.578849 16.164454,19.000003 15.887517,19 L 3.758502,18.99988 Z"/>
+ <path style="fill:#fff" d="M 3.686765,16.35231 H 1.836535 V 6.54654 L 1.092845,1.00012 h 16.8806 c 0.44463,0 0.538351,0.6154577 0.38277,1.11034 -0.178366,0.5673582 -0.393809,1.1957 -0.07685,1.1957 h 1.62779 v 1.03656 c 0,0.1424368 -0.02772,0.2924598 -0.0577,0.38849 l -1.03254,3.3078 c -0.0996,0.3190811 -0.12015,0.3383476 -0.31259,0.72438 l -2.12654,4.26582 c -0.295646,0.593063 -0.354434,1.272592 -0.37604,1.7596 -0.02611,0.588508 0.285034,1.204005 0.28312,1.753311 l -0.006,1.722079 C 16.277769,18.578849 16.092937,19.000003 15.816,19 L 3.686985,18.99988 Z"/>
```

Comment:
Can we give the number a little more breathing room by straightening up the upper left side of the state shape and shifting the lower right side a little to the right? I think the state shape would still be recognizable with these minor tweaks.

Reply:
Adjusted the right edge of the 2-digit variant; used 2-digit variant with elongated top and bottom edges to create new 3-digit variant.

I kept the left edge the same because it didn't look as recognizable to me without the angle. :shrug: 

![Screenshot from 2022-04-18 09-21-35](https://user-images.githubusercontent.com/1732117/163814372-4fcc0846-22d0-46f5-94ff-d7b129a9d1ce.png) ![Screenshot from 2022-04-18 09-21-51](https://user-images.githubusercontent.com/1732117/163814374-bc6b6a34-3b4f-4348-8c69-c40ba19282cb.png)
![Screenshot from 2022-04-18 09-22-20](https://user-images.githubusercontent.com/1732117/163814375-22b0c17e-6b7f-4ffc-85e6-2c5d2c790a60.png) ![Screenshot from 2022-04-18 09-22-35](https://user-images.githubusercontent.com/1732117/163814376-9c8971a1-4382-4e1e-82e5-a1ff4db7c82c.png)